webapp/utils/constants.jsx

```diff
@@ -865,6 +865,8 @@ export const Constants = {
     DEFAULT_MAX_NOTIFICATIONS_PER_CHANNEL: 1000,
     MAX_TEAMNAME_LENGTH: 15,
     MAX_TEAMDESCRIPTION_LENGTH: 50,
+    MIN_CHANNELNAME_LENGTH: 2,
+    MAX_CHANNELNAME_LENGTH: 15,
```

Comment:
Oh you are correct. I think I had assumed that the channel name length and the team name length are the same length. I'll update the code so that it uses the MAX_CHANNELNAME_LENGTH with length 22 and update it accordingly!